base-notebook/start-notebook.sh

```diff
@@ -4,19 +4,21 @@
 
 set -e
 
+if [[ -n "${JUPYTERHUB_API_TOKEN}" ]]; then
+    echo "WARNING: using start-singleuser.sh instead of start-notebook.sh to start a server associated with JupyterHub."
+    exec /usr/local/bin/start-singleuser.sh "$@"
+fi
+
 wrapper=""
 if [[ "${RESTARTABLE}" == "yes" ]]; then
     wrapper="run-one-constantly"
 fi
 
-if [[ -n "${JUPYTERHUB_API_TOKEN}" ]]; then
-    # launched by JupyterHub, use single-user entrypoint
-    exec /usr/local/bin/start-singleuser.sh "$@"
-elif [[ -n "${JUPYTER_ENABLE_LAB}" ]]; then
-    # shellcheck disable=SC1091
-    . /usr/local/bin/start.sh ${wrapper} jupyter lab "$@"
+if [[ -n "${JUPYTER_ENABLE_LAB}" ]]; then
+    # shellcheck disable=SC1091,SC2086
+    exec /usr/local/bin/start.sh ${wrapper} ${NOTEBOOK_ARGS} jupyter lab "$@"
 else
     echo "WARN: Jupyter Notebook deprecation notice https://github.com/jupyter/docker-stacks#jupyter-notebook-deprecation-notice."
```

Comment:
```suggestion
    echo "WARNING: Jupyter Notebook deprecation notice https://github.com/jupyter/docker-stacks#jupyter-notebook-deprecation-notice."
```

This should be WARNING, otherwise _log will not work in the future.